artemis/modules/port_scanner.py

```diff
@@ -214,7 +214,7 @@ def run(self, current_task: Task) -> None:
                     },
                 )
 
-                self.add_task(current_task, new_task)
+                self.add_task_if_domain_exists(current_task, new_task)
```

Comment:
if we have open ports, we can be sure the domain exists ;) but checking in classifier is a good idea as well as it processes incoming tasks